extensions-core/google-extensions/src/test/java/org/apache/druid/storage/google/GoogleDataSegmentKillerTest.java

```diff
@@ -99,38 +95,30 @@ public void killTest() throws SegmentLoadingException, IOException
     verifyAll();
   }
 
-  @Test(expected = SegmentLoadingException.class)
-  public void killWithErrorTest() throws SegmentLoadingException, IOException
+  @Test
+  public void killWithErrorTest() throws SegmentLoadingException
   {
-    final GoogleJsonResponseException exception = GoogleJsonResponseExceptionFactoryTesting.newMock(
-        JacksonFactory.getDefaultInstance(),
-        300,
-        "test"
-    );
-    storage.delete(EasyMock.eq(BUCKET), EasyMock.eq(INDEX_PATH));
-    EasyMock.expectLastCall().andThrow(exception);
+    Assert.assertThrows(SegmentLoadingException.class, () -> {
+      storage.delete(EasyMock.eq(BUCKET), EasyMock.eq(INDEX_PATH));
```

Comment:
nit: we can assert that the specific call throws an exception and move everything else outside the lambda:
```
Assert.assertThrows(SegmentLoadingException.class, () -> killer.kill(DATA_SEGMENT));
```

Reply:
Replaced.